app/views/listings/index.html.erb

```diff
@@ -1,2 +1,18 @@
 <h1>Listings#index</h1>
 <p>Find me in app/views/listings/index.html.erb</p>
+
+<div class="listings-container">
+  <% @listings.each do |listing| %>
+    <div class="listing">
+      <h2>
+        <%= link_to listing.title, listing %>
+      </h2>
+      <p>
+        <%= listing.user.name %>
```

Comment:
We don't have a column name called username. - I think this would need to be listing.user.vendor_name if we want to display the vendor name.
I think it would also be good to add more details about the listing for each one, so that the description and location can be viewed too on the index page.

Reply:
thanks